documentation/src/main/asciidoc/topics/ref_rest_rolling.adoc

```diff
@@ -22,7 +22,38 @@ The configuration must contain minimally the cache name, the `raw-values` set to
 host/ip of the single port in the source cluster. For details about the `remote-store` configuration,
 consult the link:https://infinispan.org/schemas/infinispan-cachestore-remote-config-12.0.xsd[XSD Schema].
 
-If the operation successfully completes, the service returns 204 (No Content).
+If the operation successfully completes, the service returns 204 (No Content). If the target cluster
+is already connected to the source cluster, it returns status 304 (Not Modified).
```

Comment:
AFAICT we only ever return 200, 404 or 500 in the method.

Reply:
ops, leftover from a previous incarnation...